.cirrus/poetry.Dockerfile

```diff
@@ -0,0 +1,34 @@
+ARG CIRRUS_AWS_ACCOUNT=275878209202
+FROM ${CIRRUS_AWS_ACCOUNT}.dkr.ecr.eu-central-1.amazonaws.com/base:j17-latest
+
+USER root
+
+ARG SCANNER_VERSION=5.0.1.3006
+ARG PYTHON_VERSION=3.12.1
+
+RUN apt-get update && apt-get install -y build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libsqlite3-dev libreadline-dev libffi-dev curl libbz2-dev grep
+RUN apt-get install -y dbus-user-session uidmap
```

Comment:
I'm not sure I'm familiar with all the packages listed here. For my understanding, how did you come up with the list of what was needed?

Reply:
`RUN apt-get install -y dbus-user-session uidmap` is not needed this is a mistake it was used for the integration tests. The first line are all dependencies needed to install python from source. https://linuxize.com/post/how-to-install-python-on-ubuntu-22-04/

Reply:
Maybe we could link this in a comment so that it's clear for whoever is not familiar with this?